layouts/community/ergodox/adnw_p_u_q/config.h

```diff
@@ -0,0 +1,10 @@
+#include QMK_KEYBOARD_H
```

Comment:
The includes should be in the keymap.c, and the define should be in the config.h.

Also you should have `#pragma once` at the top of the file here.